plugins/source/gcp/client/transformers.go

```diff
@@ -38,6 +38,10 @@ func replaceTransformer(field reflect.StructField) (string, error) {
 }
 
 func typeTransformer(field reflect.StructField) (schema.ValueType, error) {
+	if field.Name == "Etag" && field.Type == reflect.TypeOf([]byte{}) {
+		return schema.TypeString, nil
```

Comment:
```suggestion
		return schema.TypeByteArray, nil
```

Maybe we can do this until the SDK fix?